modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -126,37 +123,38 @@
 			<return type="int" />
 			<param index="0" name="var" type="Variant" />
 			<description>
-				Returns length of Variant [code]var[/code]. Length is the character count of String, element count of Array, size of Dictionary, etc.
-				[b]Note:[/b] Generates a fatal error if Variant can not provide a length.
+				Returns the length of the given Variant [param var]. The length can be the character count of a [String], the element count of any [Array] type or the size of a [Dictionary]. For every other Variant type, a run-time error is generated and execution is stopped.
```

Comment:
Technically _"any [Array] type"_ refers to `Array` and its derived types so it includes typed arrays but not packed arrays (they're different type than the linked `Array`). So from this description I could deduce I'd get a runtime error if I'd pass some packed array to it.

```suggestion
				Returns the length of the given Variant [param var]. The length can be the character count of a [String], the element count of any array type or the size of a [Dictionary]. For every other Variant type, a run-time error is generated and execution is stopped.
```